html/Task_4/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Schedule</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:400,700&display=swap&subset=latin-ext"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <input type="checkbox" id="high" />
+      <label class="label" id="high-label" for="high"
+        >Show tasks with high priority</label
```

Comment:
generally it is better to write below, because it is a bit easier to read as you separate tag from text 
```html
<label class="label" id="high-label" for="high">
  Show tasks with high priority
</label
```